GPU/Vulkan/StencilBufferVulkan.cpp

```diff
@@ -150,26 +138,39 @@ bool FramebufferManagerVulkan::NotifyStencilUpload(u32 addr, int size, bool skip
 
 	VkDescriptorSet descSet = vulkan2D_->GetDescriptorSet(overrideImageView_, nearestSampler_, VK_NULL_HANDLE, VK_NULL_HANDLE);
 
+	if (usedBits == 0) {
+		// Note: Even with skipZero, we don't necessarily start framebuffers at 0 in Vulkan.  Clear anyway.
```

Comment:
Nice, this is neat.

Reply:
Hmm, seeing it again, maybe `value` needs to be 0xFF so the shader won't discard it, d'oh.

I guess parts where the upload has zero are still a problem though, so the skipZero case probably won't work thinking this through....

-[Unknown]

Reply:
Maybe `&& stencilValue != 0`, I mean, it's a constant so shouldn't have any perf impact...

-[Unknown]

Reply:
Hm.. yeah. Or could always add another push constant to the shader to change modes...